### DISCUSSION TITLE
minio_chunk_manager failed

### DISCUSSION THREAD

**Discussion by chen982** [milvus-io/milvus#26140] [Q&A and General discussion]
There is enough space, but it shows:
[WARN] [storage/minio_chunk_manager.go:195] ["failed to put object"] [path=files/stats_log/443322260775501916/443322260775501917/443322299225148788/100/443322299225148808] [error="Storage backend has reached its minimum free drive threshold. Please delete a few objects to proceed."]

How should this happen and how to fix it?

**Comment by yhmo** [comment]
The error "Storage backend has reached its minimum free drive threshold" is returned by minio interface, it is indeed out of storage.

In this minio issue: https://github.com/minio/minio/issues/6795, you can see the minio engineer says "Minimum required in Minio is 1 gig free disk space."

**Reply by chen982** [reply]
where does it put the object to? My /home dir(where the milvus in) has 6G left and / dir(where the data storage in) has 300G left, it should have enough free space?

**Reply by yhmo** [reply]
According to the minio owner, the minio root path is not the storage path. This error is complaining about the space of the root path. And the 1GB is not configurable(the minio owner thinks it is no reason to do this).
https://github.com/minio/minio/issues/6795#issuecomment-437543165

I don't either know what is the minio root path. You can search more info from the minio community.
